src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -440,6 +440,10 @@ export const disabledListItemsAllChildren: Story = {
 };
 
 export const searchInputField: Story = {
+  args: {
+    listOverflow: "expand"
+  },
```

Comment:
This isn't being used anywhere. You can just add the prop right to the component. But I had a thought. Let's show two example: one that scrolls and one that expands. Add headings above each to explain the different configurations.